src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 20rem;
+  padding: 3.2rem 1.6rem 1.6rem;
+  border: 0.1rem solid $card-border-color;
+  border-radius: 0.5rem;
```

Comment:
It's a good idea to use `rem` but in this case it doesn't work. And for border rems usually doesn't make sense